core/src/main/java/nl/nn/adapterframework/jwt/JwtSecurityHandler.java

```diff
@@ -39,7 +39,7 @@ public JwtSecurityHandler(Map<String, Object> claimsSet, String roleClaim, Strin
 		this.principalNameClaim = principalNameClaim;
 	}
 
-	//JWTClaimNames#AUDIENCE claim may be a String or List. Others are either a String or Long (epoch date)
+	//JWTClaimNames#AUDIENCE claim may be a String or List<String>. Others are either a String or Long (epoch date)
```

Comment:
Perhaps turn the comment into a JavaDoc comment?

Reply:
Heb ik over nagedacht maar dan wordt de javadoc uit het interface overschreven en dit is niet handige/nuttige informatie voor de aanroepende partij

Reply:
Daarvoor zou je `{@inheritDoc}` kunnen gebruiken.

Maar deze interface heeft geen JavaDoc.

Maakt in ieder geval niet weel uit. Misschien zou het in de class-level JavaDoc kunnen, dat wij dit verwachten van de JWT claims?